operate/cli.py

```diff
@@ -577,38 +579,72 @@ async def _create_safes(request: Request) -> t.List[t.Dict]:
                 from_safe=False,
             )
 
-        return JSONResponse(content={"safes": safes, "message": "Safes created!"})
+        return JSONResponse(content={"safes": safes, "message": "Safes created."})
 
     @app.put("/api/wallet/safe")
     @with_retries
     async def _update_safe(request: Request) -> t.List[t.Dict]:
-        """Create wallet safe"""
+        """Update wallet safe"""
         # TODO: Extract login check as decorator
         if operate.user_account is None:
             return JSONResponse(
-                content={"error": "Cannot create safe; User account does not exist!"},
+                content={"error": "Cannot update safe; User account does not exist!"},
                 status_code=400,
             )
 
         if operate.password is None:
             return JSONResponse(
-                content={"error": "You need to login before creating a safe"},
+                content={"error": "You need to login before updating a safe."},
                 status_code=401,
             )
 
-        data = await request.json()
-        chain = Chain(data["chain"])
-        ledger_type = chain.ledger_type
         manager = operate.wallet_manager
-        if not manager.exists(ledger_type=ledger_type):
-            return JSONResponse(content={"error": "Wallet does not exist"})
+        data = await request.json()
 
-        wallet = manager.load(ledger_type=ledger_type)
-        wallet.add_or_swap_owner(
-            chain=chain,
-            owner=data.get("owner"),
+        chains = []
+        if "chain" in data:
+            if "chains" in data:
+                return JSONResponse(
+                    content={
+                        "error": "You cannot specify 'chain' and 'chains' in the same request."
+                    },
+                    status_code=401,
+                )
+            chains = [Chain[data["chain"]]]
+        elif "chains" in data:
+            chains = [Chain(chain_str) for chain_str in data["chains"]]
```

Comment:
if we just keep the `chains` field, that would be simpler right?
no matter if its length is one or more than one

Reply:
Agree, `chain` was for backwards compatibility, perhaps we should only allow `chain` and let the client make the multiple calls. cc @truemiller

Reply:
Fixed after discussion with @truemiller